parkings/forecasts/cache_parking_counts.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
```

Comment:
This file doesn't have a `if __name__ == '__main__'` part so there's no need to have a shebang either. Please remove it.